egs/swbd/s5c/local/rnnlm/tuning/run_tdnn_lstm_1e.sh

```diff
@@ -128,4 +137,19 @@ if [ $stage -le 4 ] && $run_rescore; then
   done
 fi
 
+if [ $stage -le 5 ] && $run_nbest_rescore; then
+  echo "$0: Perform nbest-rescoring on $ac_model_dir"
+  for decode_set in eval2000; do
+    decode_dir=${ac_model_dir}/decode_${decode_set}_${LM}_looped
+
+    # Lattice rescoring
+    rnnlm/lmrescore_nbest.sh \
+      --stage 6 \
```

Comment:
Since this is a chain system, I think you should be using --inv-acwt 10.0, since the acoustic scores are well calibrated (but we use a post-decode acwt of 10.0).  I don't know if this applies also to the regular non-nbest LM rescoring.

Reply:
Hmm it seems the default inv-acwt is 12 (which I copied from the original nbest scripts). I'm changing the default to 10.0